modules/sdk/project-templates/project-templates-modules-ext/src/main/resources/archetype-resources/build.gradle

```diff
@@ -1,29 +1,3 @@
-#if (${projectType} != "workspace")buildscript {
-	dependencies {
-		classpath group: "com.liferay", name: "com.liferay.gradle.plugins", version: "@com.liferay.gradle.plugins.version@"
-	}
-
-	repositories {
-		maven {
-			url "https://repository-cdn.liferay.com/nexus/content/groups/public"
-		}
-	}
-}
-
-apply plugin: "com.liferay.osgi.ext.plugin"
-
-#end
 dependencies {
-#if (${dependencyManagementEnabled} == "true")
 	originalModule group: "com.liferay", name: "${originalModuleName}"
-#else
```

Comment:
I removed prompting for originalModuleVersion so its not as confusing. It resolves with TP